docs/examples/14_stromingen_example.ipynb

```diff
@@ -309,28 +308,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# get a structured grid dataset\n",
-    "struc_ds = nlmod.get_ds(extent)\n",
-    "\n",
-    "# add x, y data to heads\n",
-    "head = head.assign_coords({\"icell2d\": head[\"icell2d\"], \"x\": ds[\"x\"], \"y\": ds[\"y\"]})\n",
-    "\n",
-    "# resample to structured grid\n",
-    "head_structured = nlmod.resample.vertex_da_to_ds(head, struc_ds)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# use imod's calculate_gxg method to calculate the GHG\n",
-    "gxg = imod.evaluate.calculate_gxg(head_structured.sel(layer=\"HLc\"))\n",
+    "# use nlmod's calculate_gxg method to calculate the GVG, GLG and GHG.\n",
+    "gxg = nlmod.evaluate.calculate_gxg(head.sel(layer=\"HLc\"))\n",
```

Comment:
Je kan de `calculate_gxg` functie ook in de gwf.output module zetten.

Reply:
Modified in commit [2ca61f7](https://github.com/ArtesiaWater/nlmod/pull/156/commits/2ca61f792c1e80de689c7f39cebf5c3c73044f06)